packages/client/src/Components/Mapping/index.jsx

```diff
@@ -161,8 +159,10 @@ const Mapping = ({ currentTurn, role, currentWargame, selectedForce, allForces,
   const clearControlledAssets = () => {
   }
 
-  const formRequestCallback = (form, payload) => {
-    console.log('Popup form requested for:', form, payload)
+  const newStateOfWorldCallback = (payload) => {
```

Comment:
` const newStateOfWorldCallback = payload => {`

Reply:
Ok.  So, if there's just one parameter, we don't need parentheses. Right?